src/CSharp/EasyMicroservice.FileManager.sln

```diff
@@ -5,13 +5,9 @@ VisualStudioVersion = 17.4.33110.190
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EasyMicroservice.FileManager", "EasyMicroservice.FileManager\EasyMicroservice.FileManager.csproj", "{FE49E974-079F-4B1F-8DE9-8DFFDB7DBD75}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "App", "App", "{1275E0F1-4EA5-4BC0-AE72-E03D4440675E}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{1275E0F1-4EA5-4BC0-AE72-E03D4440675E}"
```

Comment:
Please do not change the solution architecture, it's ok, we use different folder structures in projects.